Models/WebResource.cs

```diff
@@ -30,7 +30,8 @@ public string Name
                 {
                     var parts = value.Split('_');
                     CustomizationPrefix = parts[0];
-                    RemoteName = value.Replace($"{parts[0]}_", string.Empty);
+                    string pattern = $"^{parts[0]}_";
+                    RemoteName = Regex.Replace(value, pattern, String.Empty);
```

Comment:
Hey dude! Thanks for the update! Not to nitpick, but could you swap this to lowercase `string` for consistencies sake please?

Thanks!
```suggestion
                    RemoteName = Regex.Replace(value, pattern, string.Empty);
```